catherine_feijoo/class4/hw.rb

```diff
@@ -0,0 +1,23 @@
+
+def my_max(array)
+	if array.length == 0
+		return
+
+  elsif(array.length == 1)
+	  return array[0]
+
+  else
+  	max_val = array[0]
+  	1.upto(array.length-1) do |x|
```

Comment:
very slick :-) I love the `array.length -1` maneuver!

and *obviously* you could also do `array.each do |x|`